extensions/2.0/Khronos/KHR_materials_variants/schema/mesh.primitive.KHR_materials_variants.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "KHR_materials_variants mesh primitive extension",
+    "type": "object",
+    "allOf": [ { "$ref": "../../glTFProperty.schema.json" } ],
```

Comment:
One last little `../../` to be removed here.  (No need to re-do the ratification review document just for this).

Reply:
derp, sorry don't know how I missed this -- should be fixed now